pkg/reconciler/common/extensions_test.go

```diff
@@ -26,6 +26,14 @@ import (
 
 type TestExtension string
 
+func (t TestExtension) Manifests(v1alpha1.KComponent) ([]mf.Manifest, error) {
+	manifest, err := mf.NewManifest("testdata/kodata/additional-manifests/additional-sa.yaml")
```

Comment:
So in my custom extension, I can add new custom manifests by calling this. Only on code, right? not via Yaml?

Should we document that? Or at least capture this as a docs issue? Happy to help - I already have a use-case for this feature ;-)

Reply:
@matzew there is already a way to add custom manifests via the CRD (yaml) check @houshengbo pointers above ;)
If you want to add the manifests via code without being exposed to the user (so they cannot be subject to illegal change) you can do it with this PR.